test/integration/config_test.lua

```diff
@@ -245,6 +245,15 @@ function g.test_upload_fail()
     ]])
 end
 
+g.before_test('test_validate', function()
```

Comment:
And please add a commit where that test fails without new fixes.

Reply:
Or I suggest to extend error message: "Validation should not be called because role is disabled"